src/core/Akka.API.Tests/verify/CoreAPISpec.ApprovePersistenceSqlCommon.Core.verified.txt

```diff
@@ -91,9 +91,6 @@ namespace Akka.Persistence.Sql.Common.Journal
         protected virtual string ByPersistenceIdSql { get; }
         protected virtual string ByTagSql { get; }
         protected virtual string DeleteBatchSql { get; }
-        protected bool HasNewEventsSubscribers { get; }
```

Comment:
No longer need these defined on the journal implementations. Could add these back to be non-breaking, but they're obsolete now regardless and not used anywhere inside the journal.